src/components/SelectableCardField/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import { Meta, Story } from '@storybook/react'
+import React, { ComponentProps } from 'react'
+import SelectableCardField from '..'
+import mockErrors from '../../../mocks/mockErrors'
+import Form, { FormProps } from '../../Form'
+import Submit from '../../Submit'
+
+export default {
+  component: SelectableCardField,
+  decorators: [ChildStory => <Form errors={mockErrors}>{ChildStory()}</Form>],
+  title: 'Components/Fields/SelectableCardField',
+} as Meta
+
+const Template: Story<ComponentProps<typeof SelectableCardField>> = args => (
+  <SelectableCardField {...args}>Radio</SelectableCardField>
+)
+
+export const Default = Template.bind({})
+
+Default.args = {
+  name: 'example',
+}
+
+export const Checked: Story<FormProps> = ({ errors }) => (
+  <Form errors={errors} initialValues={{ foo: 'bar' }}>
+    <div style={{ display: 'flex', gap: '16px' }}>
+      <SelectableCardField name="foo" value="bar">
+        Radio left
+      </SelectableCardField>
+      <SelectableCardField name="foo" value="barbar">
```

Comment:
`name` should be different I think?

Reply:
@QuiiBz nope, name should be same as they both change same variable into the form :)